### ISSUE TITLE
Format code by improving the indentation in blog.css stylesheet

### ISSUE THREAD

**Issue by simransia** [publiclab/plots2#11006]
### 💡Format this Issue Into a First Timers Issue
This has been marked as a good candidate for becoming a [first-timers-only issue](https://code.publiclab.org/#r=all) like these, meaning that it's **simple, self-contained**, and with some extra formatting✍️, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider **reformatting it as a first-timers-only issue**, and then ping @publiclab/reviewers to get it labelled🏷. Or, if this is not your first time, **try to solve it yourself**🧑‍💻!

****

Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [plots2/app/assets/stylesheets/blog.css](https://github.com/publiclab/plots2/blob/main/app/assets/stylesheets/blog.css) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -1,60 +1,59 @@
 .blog-header h1 {
-    font-size:4em;
-    margin-top:100px;
-    color:#326;
-    line-height:1.3em
-  }
-
+  font-size: 4em;
+  margin-top: 100px;
+  color: #326;
+  line-height: 1.3em
+}
+
+.blog h1 {
+  text-align: center;
+  padding: 10px 40px;
+}
+
+@media (max-width:400px) {
   .blog h1 {
-    text-align:center;
-    padding:10px 40px;
-  }
-
-  @media (max-width:400px) {
-    .blog h1 {
-      padding:10px;
-    }
-  }
-
-  .blog h1 a {
-   clear:left;
-   color:#444;
-   line-height:1.2em;
-  }
-
-  .blog .blog-entry img {
-    max-width: 100%;
-  }
-
-  .blog .meta {
-    text-align:center;
-    margin:10px 0 20px;
-  }
-
-  .blog a.img {
-    display:block;
-    text-align:center;
-    margin-bottom:20px;
-  }
-
-  .blog-header {
-    text-align:center;
-  }
-
-  .blog-header p {
-    color:#444;
-  }
-
-  .blog {
-    font-size:15px;
-  }
-
-  .blog p {
-    margin-bottom:20px;
-  }
-
-  .blog p img {
-    display: block;
-    margin: 0 auto
-  }
-

+    padding: 10px;
+  }
+}
+
+.blog h1 a {
+  clear: left;
+  color: #444;
+  line-height: 1.2em;
+}
+
+.blog .blog-entry img {
+  max-width: 100%;
+}
+
+.blog .meta {
+  text-align: center;
+  margin: 10px 0 20px;
+}
+
+.blog a.img {
+  display: block;
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.blog-header {
+  text-align: center;
+}
+
+.blog-header p {
+  color: #444;
+}
+
+.blog {
+  font-size: 15px;
+}
+
+.blog p {
+  margin-bottom: 20px;
+}
+
+.blog p img {
+  display: block;
+  margin: 0 auto
+}

```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

**Comment by simransia** [comment]
Hello @TildaDares @cesswairimu @jywarren . Please go through this issue and check if this issue is eligible for becoming an FTO issue for new contributors.

Thank You❤️

**Comment by TildaDares** [comment]
Thanks for opening this issue @simransia. I took a look at the file but I don’t see any indentation problems.

**Comment by simransia** [comment]
Yes mam, my apologies. There is no need for fixing indentation in that file. Actually in vs code the file was not looking properly formatted, so I had created the issue.

I have updated this issue, please review it again @TildaDares.

Thankyou

**Comment by TildaDares** [comment]
Hi @simransia, indentations are supposed to be two spaces deep.

**Comment by simransia** [comment]
Hello @TildaDares , I have again made some corrections related to indentation. I think now it is up to the expectation.
Please review it.

Thank You 💛

**Comment by TildaDares** [comment]
This looks great @simransia. Thank you!!

**Comment by TildaDares** [comment]
Reserved for @anirudh-t-d for 24 hours!!

**Comment by anirudh-t-d** [comment]
Hi @TildaDares thank you for assigning an fto to me!!! I will work on this ASAP

**Comment by rajvidg** [comment]
@simransia @TildaDares is this issue still open or assigned to someone, please let me know, I would like to help if possible.

**Comment by AmineZCS** [comment]
@anirudh-t-d @TildaDares @rajvidg  is this issue still open? I would like to make my first contribute

**Comment by anirudh-t-d** [comment]
I am working on it.

**Comment by TildaDares** [comment]
HI @anirudh-t-d, do you need help with this?

**Comment by anirudh-t-d** [comment]
Sorry mam. I couldn't get time to devote on this as I was on a medical emergency. I will complete it by Saturday. Apologies again for this

**Comment by TildaDares** [comment]
Hi @anirudh-t-d, please take all the time you need. Thank you!

**Comment by nawed2611** [comment]
Hey! Is this issue still open?

**Comment by TildaDares** [comment]
Hi @nawed2611, someone’s working on this issue. You can leave a comment in https://github.com/publiclab/plots2/issues/11105 and we’ll look for an issue for you. Thanks!!

**Comment by TildaDares** [comment]
Hi @anirudh-t-d, any updates?

**Comment by anirudh-t-d** [comment]
Hi @TildaDares  can you please review my pull request if everything is okay?

**Comment by TildaDares** [comment]
Hi @anirudh-t-d, I can't find your PR.

**Comment by TildaDares** [comment]
Hi @anirudh-t-d, you need to make a PR in publiclab/plots2.

**Comment by stirlhoss** [comment]
@TildaDares PR submitted, ready for review.
